gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -0,0 +1,375 @@
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_clinvar.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Inferred sample populations
+pop = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_pop.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+
+def pop_tsv_path(version: str = CURRENT_VERSION) -> str:
+    """
+    Path to tab delimited file indicating inferred sample populations.
+
+    :param version: gnomAD Version
+    :return: String path to sample populations
+    """
+    return f"gs://gnomad/v{version}/sample_qc/temp/exomes/gnomad_v{version}_RF_pop_assignments.txt.gz"
+
+
+# TODO: Should this still be a "temp" directory?
+
+
+def pop_rf_path(version: str = CURRENT_VERSION) -> str:
+    """
+    Path to RF model used for inferring sample populations.
+
+    :param version: gnomAD Version
+    :return: String path to sample pop RF model
+    """
+    return f"gs://gnomad/v{version}/sample_qc/temp/exomes/gnomad_v{version}_pop.RF_fit.pickle"
+
+
+def _import_related_samples_to_drop(**kwargs):
+    ht = hl.import_table(**kwargs)
+    ht = ht.key_by(s=ht.f0)
+
+    return ht
+
+
+# Hard-filtered samples
+hard_filtered_samples = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_hard_filtered_samples.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Results of running population-based metrics filtering
+# For v4 might not use this (regresed metrics used instead)
+stratified_metrics = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_stratified_metrics.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Results of running regressed metrics filtering
+regressed_metrics = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_regressed_metrics.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Ranking of all samples based on quality metrics. Used to remove relateds for PCA.
+pca_samples_rankings = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_pca_samples_ranking.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Ranking of all release samples based on quality metrics. Used to remove relateds for release.
+release_samples_rankings = VersionedTableResource(
+    CURRENT_VERSION,
+    {
+        release: TableResource(
+            f"{get_sample_qc_root(release)}/gnomad_v{release}_release_samples_ranking.ht"
+        )
+        for release in VERSIONS
+    },
+)
+
+# Picard metrics
```

Comment:
We will probably change this to just have the picard metrics on the project metadata HT, so maybe move this to the meta.py resource file